packages/lib/src/components/internal/BaseElement/BaseElement.ts

```diff
@@ -8,6 +8,7 @@ import type { ICore } from '../../../core/types';
 import type { BaseElementProps, IBaseElement } from './types';
 import type { PaymentData } from '../../../types/global-types';
 import type { AnalyticsInitialEvent, SendAnalyticsObject } from '../../../core/Analytics/types';
+import { off, on } from '../SecuredFields/lib/utilities/dom';
```

Comment:
We could move these functions out of the SecuredFields folder and place somewhere more top-level, since they are being used not only in the secured fields context anymore.

Reply:
Was on my TODO list. Now it's done!

Reply:
Done